examples/seismic/self_adjoint/time_blocking.ipynb

```diff
@@ -0,0 +1,965 @@
+{
```

Comment:
<p>I wouldn't print them, otherwise you'll need again an NBVAL_IGNORE because depending on the platform/compiler/etc there might be some fp delta</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/devitocodes/devito/pull/1385/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
I dont print anymore, just a summary of maximum error at the end. Added IGNORE, and an assert for the difference small.